### DISCUSSION TITLE
helptags for unloaded plugins

### DISCUSSION THREAD

**Discussion by folke** [ibhagwan/fzf-lua#1296] [Ideas]
Would be great if I could somehow hook into the `helptags` to also show help from plugins that are not yet loaded and as such not yet on the rtp.

For lazy, this would be the paths below:

```lua
local lazy = package.loaded["lazy.core.config"]
if lazy then
  local paths = {}
  for _, plugin in pairs(lazy.plugins) do
    if not plugin._.loaded then
      table.insert(paths, plugin.dir)
    end
  end
end
```

What would be the best way to make this possible?

https://github.com/ibhagwan/fzf-lua/blob/main/lua/fzf-lua/providers/helptags.lua#L36

Edit, even shorter:

```lua
local lazy = package.loaded["lazy.core.util"]
if lazy then
  local paths = lazy.get_unloaded_rtp("")
end
```

**Comment by ibhagwan** [comment]
Great idea @folke, this always bothered me when I couldn’t find help of lazy loaded plugins

https://github.com/ibhagwan/fzf-lua/commit/3b070d0cccf9aeecc4d2d9fb376849c6130b90ec

**Reply by folke** [reply]
That's awesome, thanks! :)

**Reply by ibhagwan** [reply]
Ty @folke for improving our neovim experience anywhere you set foot ❤️
